Include/internal/pycore_unicodeobject.h

```diff
@@ -11,6 +11,230 @@ extern "C" {
+extern void _PyUnicode_SET_UTF8_LENGTH(PyObject *op, Py_ssize_t length);
+extern PyObject* _PyUnicode_FromUCS1(const Py_UCS1* u, Py_ssize_t size);
+extern PyObject* _PyUnicode_TranslateCharmap(
+    PyObject *input,
+    PyObject *mapping,
+    const char *errors);
+extern int _PyUnicode_FillUTF8(PyObject *unicode);
+extern int _PyUnicode_DecodeUTF8Writer(
+    _PyUnicodeWriter *writer,
+    const char *s,
+    Py_ssize_t size,
+    _Py_error_handler error_handler,
+    const char *errors,
+    Py_ssize_t *consumed);
+extern int _Py_normalize_encoding(const char *, char *, size_t);
+extern void* _PyUnicode_AsKind(
+    int skind,
+    void const *data,
+    Py_ssize_t len,
+    int kind);
+extern int _PyUnicode_Tailmatch(
+    PyObject *self,
+    PyObject *substring,
+    Py_ssize_t start,
+    Py_ssize_t end,
+    int direction);
+extern Py_ssize_t _PyUnicode_Count(
+    PyObject *str,
+    PyObject *substr,
+    Py_ssize_t start,
+    Py_ssize_t end);
+extern PyObject * _PyUnicode_Replace(
+    PyObject *self,
+    PyObject *str1,
+    PyObject *str2,
+    Py_ssize_t maxcount);
+extern Py_ssize_t _PyUnicode_AnylibFindSlice(
+    PyObject* s1,
+    PyObject* s2,
+    Py_ssize_t start,
+    Py_ssize_t end,
+    int direction);
+extern int _PyUnicode_FindMaxCharSurrogates(
+    const wchar_t *begin,
+    const wchar_t *end,
+    Py_UCS4 *maxchar,
+    Py_ssize_t *num_surrogates);
+extern void _PyUnicode_WriteWideChar(
+    int kind,
+    void *data,
+    const wchar_t *u,
+    Py_ssize_t size,
+    Py_ssize_t num_surrogates);
+extern PyObject* _PyUnicode_FromUCS2(const Py_UCS2 *u, Py_ssize_t size);
+extern PyObject* _PyUnicode_FromUCS4(const Py_UCS4 *u, Py_ssize_t size);
+extern PyObject* _PyUnicode_FromOrdinal(Py_UCS4 ordinal);
+extern PyObject* _PyUnicode_do_string_format(
+    PyObject *self,
+    PyObject *args,
+    PyObject *kwargs);
+extern PyObject* _PyUnicode_do_string_format_map(
+    PyObject *self,
+    PyObject *obj);
+extern Py_hash_t _PyUnicode_Hash(PyObject *self);
+extern PyObject* _PyUnicode_Iter(PyObject *seq);
+extern int _PyUnicode_IsModifiable(PyObject *unicode);
+extern void _PyUnicode_Fill(
+    int kind,
+    void *data,
+    Py_UCS4 value,
+    Py_ssize_t start,
+    Py_ssize_t length);
+extern PyObject* _PyUnicode_ResizeCompact(
+    PyObject *unicode,
+    Py_ssize_t length);
+extern int _PyUnicode_CheckModifiable(PyObject *unicode);
+extern PyObject* _PyUnicode_Repr(PyObject *unicode);
+extern PyObject* _PyUnicode_Pad(
+    PyObject *self,
+    Py_ssize_t left,
+    Py_ssize_t right,
+    Py_UCS4 fill);
+extern int _PyUnicode_Resize(PyObject **p_unicode, Py_ssize_t length);
+extern PyObject* _PyUnicode_EncodeUTF8(
+    PyObject *unicode,
+    _Py_error_handler error_handler,
+    const char *errors);
+extern PyObject* _PyUnicode_DecodeUTF8(
+    const char *s,
+    Py_ssize_t size,
+    _Py_error_handler error_handler,
+    const char *errors,
+    Py_ssize_t *consumed);
+extern char* _Py_backslashreplace(
```

Comment:
Can we have something like `_Py_[something else]_backslashreplace`? Otherwise, if I write `_Py_*` I'll get this one as being possibly autocompleted.